src/sentry/tasks/integrations.py

```diff
@@ -30,3 +30,22 @@ def post_comment(external_issue_id, data, **kwargs):
 @retry(on=(IntegrationError,))
 def sync_metadata(installation):
     installation.sync_metadata()
+
+
+@instrumented_task(
+    name='sentry.tasks.integrations.sync_assignee_outbound',
+    queue='integrations',
+    default_retry_delay=60 * 5,
+    max_retries=5
+)
+@retry(exclude=(ExternalIssue.DoesNotExist, Integration.DoesNotExist, User.DoesNotExist))
+def sync_assignee_outbound(external_issue_id, user_id, **kwargs):
+    # sync Sentry assignee to an external issue
+    external_issue = ExternalIssue.objects.get(id=external_issue_id)
+    integration = Integration.objects.get(id=external_issue.integration_id)
+    # assume unassign if None
```

Comment:
This seems... not the best API. 👀

Reply:
yeah 😐 it is what we do here: https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/project_group_index.py#L717

what do you think would be better? the task and/or the installation method below could take a kwarg like `deassign=True`? I could also break this up into two tasks -- i just don't think i'd want to do that for the installation method since each integration will be implementing that